client/src/components/Logo.js

```diff
@@ -1,12 +1,22 @@
 import styled from "styled-components";
+import { bool, func } from "prop-types";
```

Comment:
Nice, didn't know they can be import that way!

Reply:
auto-import :)